static/js/chart/draw_d3_map.ts

```diff
@@ -461,7 +466,20 @@ export function drawD3Map(
     "click",
     onMapClick(canClickRegion, containerElement, redirectAction)
   );
-
+  // If provided, draw border of containing place
```

Comment:
instead of adding more to this already very big function, I think it would be better to just call addGeoJsonLayer from d3_map.tsx & we can add a `getRegionBorder: (geoDcid: string) => string` argument to the `addpolygonLayer` function

Reply:
Moved the logic to d3_map.tsx using `addPolygonLayer` instead. Also had to add an extra "allowMouseover" argument to addPolygonLayer, otherwise the entire map gets highlighted on mouseover.